detect-nuget-inspector/detect-nuget-inspector/Inspection/Inspectors/SolutionInspector.cs

```diff
@@ -106,7 +121,31 @@ public Container GetContainer()
                         try
                         {
                             string projectRelativePath = project.Path;
-                            string projectPath = PathUtil.Combine(solutionDirectory, projectRelativePath);
+                            string projectPath = null;
```

Comment:
This is a very weird situation which required a fix when the there are multiple props file which are mentioned under sln file ProjectSection, as an example 
`ProjectSection(SolutionItems) = preProject
		..\samples\Directory.Build.props = ..\samples\Directory.Build.props
		..\samples\Directory.Packages.props = ..\samples\Directory.Packages.props
EndProjectSection`.
This is a different props file which was present under samples folder which is not in the root directory for sln file, so this logic would go up each directory to check if props file exists in that directory and parse it if it does.